### PR TITLE
Adds prefab: DreamPlaza Mall

### PR DESCRIPTION
**Pull request by WALPVRGIS**
[FEATURE] [MAPPING] [SPRITES]

## About the PR 
This adds the DreamPlaza Mall, a [fragment of an] 80's-esque space mall-station. I intended this prefab to be a Cool Hang Out and Gimmick Place. It features a mini food court, an abandoned clothing store, maybe a little bit of storytelling (environmental and otherwise).

This is for the "April" Monthly Contest here; [https://forum.ss13.co/showthread.php?tid=16992](https://forum.ss13.co/showthread.php?tid=16992)

**- Size:** 30x30
**- Probability:** 30
**- Features:** _D E L U X E_, a new clothing store (albeit a bit wrecked).
**- Hazards:** A very angry lil Beepsky with a new look. And environmental hazards.
**- New Loot:** A recoloured space suit + new style of space helmet, plus two backpacks not available elsewhere (angel backpack and white studded backpack).
**- Other Loot:** Some cash, some drugs, lots of trash because there is a storage room and its a Mall.
**- Other New Things:** Bunch of new furniture and decor.
**- Things to consider:** Ice Cream Bar in the food court has a Kitchen Dispenser/Fountain and a Foodtech, as well as oven and glass recycler. The clothing store has a Clothing Booth, as I wanted a clothing store potentially available for space maps that aren't Donut 3. The angel backpack can not be turned into a satchel, as it looked silly and defeats the purpose of the backpack looking like wings!

Some ingame screenshots;
dreamplaza_0
dreamplaza_1
dreamplaza_2
GIF 20-09-2021 16-38-23

## Why's this needed? 
I'm submitting this for the monthly contest! Also I really wanted a clothing store prefab, and a retro mall prefab so.. I combined it into one. This has a lot of RP and Gimmick potential I think...!

## Changelog

```changelog
(u)Walpvrgis
(*)Added a new space prefab; the DreamPlaza Mall (or what is left of it).
```

### CODE REVIEW

**Comment by vocalpocal** [comment]
This looks very nice!

**Comment by Tarmunora** [comment]
s t y l i n

File: assets/maps/prefabs/prefab_dreamplaza.dmm [hunk 1]
```diff
@@ -0,0 +1,259 @@
+"ae" = (/obj/decal/poster/wallsign/poster_clown{pixel_x = 7; pixel_y = 6},/obj/decal/cleanable/gangtag{pixel_x = -4; pixel_y = -1},/turf/simulated/wall/auto/jen/dark1,/area/prefab/dreamplaza)
```
Comment (Sovexe) [inline assets/maps/prefabs/prefab_dreamplaza.dmm:1]: convert map to TGM from DMM

Comment (Sovexe) [inline assets/maps/prefabs/prefab_dreamplaza.dmm:1]: preferable  to have maps in TGM format

**Pull request merged**